tests/test_enum_types.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+#
+# A library that provides a Python interface to the Telegram Bot API
+# Copyright (C) 2015-2023
+# Leandro Toledo de Souza <devs@python-telegram-bot.org>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser Public License for more details.
+#
+# You should have received a copy of the GNU Lesser Public License
+# along with this program.  If not, see [http://www.gnu.org/licenses/].
+import re
+from pathlib import Path
+
+telegram_root = Path(__file__).parent.parent / "telegram"
+telegram_ext_root = telegram_root / "ext"
+exclude_dirs = {
+    # We touch passport stuff only if strictly necessary.
+    telegram_root
+    / "_passport",
```

Comment:
```suggestion
    telegram_root / "_passport",
```
having this on a new line kinda confused me (specially since I'm reviewing on mobile)

Reply:
black wants it that way …